sdk/data/azcosmos/cosmos_http_constants.go

```diff
@@ -78,6 +78,7 @@ const (
 	cosmosHeaderIsPartitionKeyDeletePending        string = "x-ms-cosmos-is-partition-key-delete-pending"
 	cosmosHeaderQueryExecutionInfo                 string = "x-ms-cosmos-query-execution-info"
 	headerXmsItemCount                             string = "x-ms-item-count"
+	headerDedicatedGatewayMaxAge                   string = "x-ms-dedicatedgateway-max-age"
```

Comment:
Is this the correct header value, was unable to find it in the dotnet SDK for example but I see it in the Python and Java SDKs

Reply:
Looks like it. For .NET it's in the Direct NuGet package that comes from our internal code.